package.json

```diff
@@ -22,13 +22,9 @@
   },
   "keywords": [
     "UBP",
-    "CRS",
-    "travel",
-    "toma",
-    "cets",
-    "sabre"
+    "CRS"
   ],
-  "author": "Steve Reichenbach",
+  "author": "FTI",
```

Comment:
Well, technically, you are still the author.

Reply:
Yes, but I saw it on another repo of tourstream and thought: "that could be more correct".